Sources/Request/Request.swift

```diff
@@ -4,16 +4,20 @@ public struct Request {
 
      public let method: String
      public let path: String
+     public let http: String
      public var queries: [String: String] 
      public var body: [String: String] 
      public var cookie: String  
+     public var headers: [String: String]
 
-     public init(method: String, path: String, queries: [String: String] = ["fname": "World"], body: [String: String] = [String(): String()], cookie: String = String()) {
+     public init(method: String, path: String, http: String = String(), queries: [String: String] = ["fname": "World"], body: [String: String] = [:], cookie: String = String(), headers: [String: String] = [:]) {
```

Comment:
As your server gets abstracted a bit, I don't really like "fname" (something that's route specific being passed in as a default value. 

Also, why do all of these have default values? To me a value is either optional or it isn't, for example I could see `cookie` being null, where every HTTP request should have a `http` (which seems to represent the http version.)